tutorials/boundary_conditions.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<p>I don't think this is exactly what I understood to be the plan. That's fine, but I suggest:</p><p> </p><p>To set a boundary condition do the following:</p><p>1) Specify the type of the boundary condition (Dirichlet, Neumann or Robin) in the method named <code>bc_type_&lt;flux_name&gt;</code>.</p><p>2) Set the values of relevant physical quantities on the boundary in the method named <code>bc_values_&lt;physical_quantity&gt;</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>